### DISCUSSION TITLE
How to use Reqnroll.MsTest.2.1.2-local.nupkg?

### DISCUSSION THREAD

**Discussion by drosenba** [reqnroll/Reqnroll#292] [General]
I downloaded the Reqnroll-main repo and did a local Build.  I see it generates nuget packages in 
...\Reqnroll-main\GeneratedNuGetPackages\Release
In a separate solution\project (.NET 8) that was working with the ReqnRoll.MsTest package from nuget.org, I removed it and then referenced Reqnroll.MsTest.2.1.2-local.nupkg in above folder.  When I try to Build the project, I get error

Error	MSB4062	The "Reqnroll.Tools.MsBuild.Generation.ReplaceTokenInFileTask" task could not be loaded from the assembly C:\...\.nuget\packages\reqnroll.tools.msbuild.generation\2.1.2-local\build\..\tasks\net462\Reqnroll.Tools.MsBuild.Generation.dll.
Could not load file or assembly 'file:///C:\...\.nuget\packages\reqnroll.tools.msbuild.generation\2.1.2-local\tasks\net462\Reqnroll.Tools.MsBuild.Generation.dll' or one of its dependencies.
The system cannot find the file specified. Confirm that the <UsingTask> declaration is correct, that the assembly and all its dependencies are available, and that the task contains a public class that implements Microsoft.Build.Framework.ITask.

I can see that it's trying to reference subfolder net462, but I'm not sure why or what I should do.

**Comment by clrudolphi** [comment]
The first step is to force a rebuild of the Reqnroll.Tools.MsBuild.Generation project. Then rebuild Reqnroll.

Following the above depends on how you are consuming your local build.
Is your test project directly consuming the Reqnroll assembly (DLL) via a project reference or is it consuming the locally produced nuget package?

If consuming the local nuget package, then I suggest that once you build Reqnroll, go to your local nuget package cache and delete the prior build nuget packages for Reqnroll. The location for this is (on Windows) 'C:\Users\\<you>\\.nuget\packages'

Delete the subfolders for reqnroll, reqnroll.mstest, and reqnroll.tools.msbuild.generation

Then open your test solution, Clean; Restore Nuget Packages, Rebuild

That should result in your test package pulling in your most recent local build of Reqnroll.
